spec/index.html

```diff
@@ -367,6 +367,39 @@ <h2>Introduction</h2>
       <p>There is also a [[[SPARQL12-RESULTS-XML]]] which follows a similar design pattern but uses XML as the
       serialization.</p>
       <p>Unless otherwise noted in the section heading, all sections and appendices in this document are normative.</p>
+
+	  <section id="version-announcement">
+	    <h3>Version Announcement</h3>
+	    <p>To indicate that RDF 1.2 features are used in the SPARQL results,
```

Comment:
This can be read that is `version="1.2"` is used, then there are 1.2 features in the results. I think the intention is to indicate the possibility.

I would suggest s/are used/may be present/ ... but we get into "may"/"MAY".